tools/pg_tools/pg-query.xml

```diff
@@ -0,0 +1,48 @@
+<tool id="pg_query" name="Postgresql" version="@PG_VERSION@">
+    <description>import sql dump</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="aggressive"><![CDATA[
+            @UNTAR_INFILE@ &&
+            @PG_START@ &&
+            ls -l ./postgresql
+            &&
+            @PSQL@ -c '$query' -L logfile.log > '$outfile' &&
+            @PG_STOP@ &&
+            cat logfile.log
+    ]]>
+    </command>
+    <inputs>
+        <param format="postgresql" name="infile" type="data" label="Input database" />
+        <param name="query" type="text" area="True" size="5x50" label="The query to be issued to the database">
+            <sanitizer>
+                <valid initial="string.letters,string.digits">
```

Comment:
this won't be enough, we'll need `'` at least.

Reply:
fixed, other can be at any time.